README.rst

```diff
@@ -15,12 +15,14 @@ Setup
 
 Start by `creating a project in the Google Developers Console <https://console.developers.google.com>`_.
 
+In the **Authorized JavaScript origins** add the Root URL for your installation::
```

Comment:
Also unclear on my side, added it as I assume it was a new feature or field between when this Readme was created or an oversight.  I can remove it if you think it's superfluous!